core/CliMulti/Process.php

```diff
@@ -210,7 +210,7 @@ public static function isSupported()
             return false;
         }
 
-        if (!self::commandExists('ps') || !self::returnsSuccessCode('ps') || !self::commandExists('awk')) {
+        if (!self::returnsSuccessCode('ps --version') || !self::returnsSuccessCode('awk --version')) {
```

Comment:
fyi haven't tested it but locally on my Mac I get this error:

```
 ps --version
ps: illegal option -- -
usage: ps [-AaCcEefhjlMmrSTvwXx] [-O fmt | -o fmt] [-G gid[,gid...]]
          [-g grp[,grp...]] [-u [uid,uid...]]
          [-p pid[,pid...]] [-t tty[,tty...]] [-U user[,user...]]
       ps [-L]
```

so this doesn't seem to work on mac.

Reply:
I guess we can just use `ps`/`awk` then.